Content/Items/BarrierDye/BaseBarrierDye.cs

```diff
@@ -62,7 +62,7 @@ public override void PreDrawEffects(SpriteBatch spriteBatch, Player Player)
 			BarrierPlayer barrier = Player.GetModPlayer<BarrierPlayer>();
 
 			spriteBatch.End();
-			spriteBatch.Begin(default, BlendState.Additive, default, default, default, default, Main.GameViewMatrix.TransformationMatrix);
+			spriteBatch.Begin(default, BlendState.Additive, default, default, default, default, Main.GameViewMatrix.ZoomMatrix);
```

Comment:
Is there a reason to switch back to zoom matrix only over transformation matrix? Could this cause issues with mod compat?